Doppler.Sap/Factory/BillingRequestHandler.cs

```diff
@@ -81,7 +81,7 @@ public async Task<SapTaskResult> Handle(SapTask dequeuedTask)
                     IsSuccessful = false,
                     SapResponseContent = ex.Message,
                     TaskName = "Creating Billing Request",
-                    IdUser = dequeuedTask.BillingRequest.UserId.ToString()
+                    IdUser = dequeuedTask.BillingRequest?.UserId.ToString()
```

Comment:
Why you add **?** ?

Reply:
Just in case `BillingRequest `is `null`.

Reply:
That it's in Catch, maybe if there is an exception can happend....you think that it's not neccessary?